sympy/matrices/common.py

```diff
@@ -2019,6 +2019,67 @@ def replace(self, F, G, map=False, simultaneous=True, exact=None):
         return self.applyfunc(
             lambda x: x.replace(F, G, map=map, simultaneous=simultaneous, exact=exact))
 
+    def rot90(self, k=1):
+        """Rotates Matrix by 90 degrees
+   
+        Parameter k specifies the rotation
```

Comment:
You should follow other examples in sympy docstring guide, to denote a parameter

Reply:
Ohhh alright 😄 ... will do that

Reply:
Done